Allow users to view the recovery phrase again when creating a profile for an empty keycard

Parveshdhull opened the pull request:
fixes https://github.com/status-im/status-mobile/issues/22002

status: ready

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (25)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 16eb2a96 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22009/1/) | 2025-02-03 08:14:41 | ~5 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250203-080922-16eb2a-pr22009-tests.log) |
| :heavy_check_mark: | 16eb2a96 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22009/1/) | 2025-02-03 08:18:40 | ~9 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250203-080917-16eb2a-pr22009-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250203-080917-16eb2a-pr22009-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 16eb2a96 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22009/1/) | 2025-02-03 08:19:07 | ~9 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250203-080922-16eb2a-pr22009-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250203-080922-16eb2a-pr22009-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 16eb2a96 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22009/1/) | 2025-02-03 08:21:09 | ~11 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/xEkUyj) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FxEkUyj&size=400x400&errorCorrectionLevel=L)|
| | | | | | | |
| :heavy_check_mark: | 0767bb8e | [#3](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22009/3/) | 2025-02-03 11:50:54 | ~4 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250203-114615-0767bb-pr22009-tests.log) |
| :heavy_check_mark: | 0767bb8e | [#3](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22009/3/) | 2025-02-03 11:53:48 | ~7 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250203-114615-0767bb-pr22009-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250203-114615-0767bb-pr22009-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 0767bb8e | [#3](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22009/3/) | 2025-02-03 11:54:37 | ~8 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250203-114615-0767bb-pr22009-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250203-114615-0767bb-pr22009-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 0767bb8e | [#3](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22009/3/) | 2025-02-03 11:57:47 | ~11 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/auKVxp) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FauKVxp&size=400x400&errorCorrectionLevel=L)|
| | | | | | | |
| :heavy_check_mark: | 134c3491 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22009/4/) | 2025-02-03 12:06:17 | ~6 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250203-120005-134c34-pr22009-tests.log) |
| :heavy_check_mark: | 134c3491 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22009/4/) | 2025-02-03 12:07:37 | ~7 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250203-120000-134c34-pr22009-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250203-120000-134c34-pr22009-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 134c3491 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22009/4/) | 2025-02-03 12:08:57 | ~8 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250203-120000-134c34-pr22009-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250203-120000-134c34-pr22009-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 134c3491 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22009/4/) | 2025-02-03 12:11:49 | ~11 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/jpNZAs) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FjpNZAs&size=400x400&errorCorrectionLevel=L)|
| | | | | | | |
| :x: | b21903c0 | [#5](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22009/5/) | 2025-02-03 14:33:24 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22009/5/consoleText) |
| | | | | | | |
| :heavy_check_mark: | 64a52f6e | [#6](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22009/6/) | 2025-02-03 14:42:43 | ~4 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250203-143816-64a52f-pr22009-tests.log) |
| :heavy_check_mark: | 64a52f6e | [#6](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22009/6/) | 2025-02-03 14:45:21 | ~7 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250203-143812-64a52f-pr22009-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250203-143812-64a52f-pr22009-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 64a52f6e | [#6](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22009/6/) | 2025-02-03 14:46:25 | ~8 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250203-143812-64a52f-pr22009-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250203-143812-64a52f-pr22009-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 64a52f6e | [#6](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22009/6/) | 2025-02-03 14:48:55 | ~10 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/LhLSAn) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FLhLSAn&size=400x400&errorCorrectionLevel=L)|
| | | | | | | |
| :heavy_check_mark: | 525db13e | [#7](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22009/7/) | 2025-02-04 09:54:29 | ~4 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250204-094931-525db1-pr22009-tests.log) |
| :heavy_check_mark: | 525db13e | [#7](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22009/7/) | 2025-02-04 09:58:27 | ~8 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250204-094928-525db1-pr22009-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250204-094928-525db1-pr22009-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 525db13e | [#7](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22009/7/) | 2025-02-04 09:58:58 | ~9 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250204-094931-525db1-pr22009-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250204-094931-525db1-pr22009-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 525db13e | [#7](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22009/7/) | 2025-02-04 10:01:12 | ~11 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/1Fzcf6) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2F1Fzcf6&size=400x400&errorCorrectionLevel=L)|
| | | | | | | |
| :heavy_check_mark: | 16a92863 | [#9](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22009/9/) | 2025-02-04 16:08:39 | ~4 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250204-160345-16a928-pr22009-tests.log) |
| :heavy_check_mark: | 16a92863 | [#9](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22009/9/) | 2025-02-04 16:11:28 | ~7 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250204-160341-16a928-pr22009-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250204-160341-16a928-pr22009-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 16a92863 | [#9](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22009/9/) | 2025-02-04 16:12:06 | ~8 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250204-160345-16a928-pr22009-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250204-160345-16a928-pr22009-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 16a92863 | [#9](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22009/9/) | 2025-02-04 16:16:48 | ~13 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/VL33q7) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FVL33q7&size=400x400&errorCorrectionLevel=L)|
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | daaf27b1 | [#10](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22009/10/) | 2025-02-05 12:42:25 | ~11 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250205-123109-daaf27-pr22009-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250205-123109-daaf27-pr22009-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | daaf27b1 | [#10](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22009/10/) | 2025-02-05 12:42:45 | ~11 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/myNwNK) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FmyNwNK&size=400x400&errorCorrectionLevel=L)|
| :x: | daaf27b1 | [#10](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22009/10/) | 2025-02-05 12:44:32 | ~13 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22009/10/consoleText) |
| :heavy_check_mark: | daaf27b1 | [#10](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22009/10/) | 2025-02-05 12:44:42 | ~13 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250205-123113-daaf27-pr22009-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250205-123113-daaf27-pr22009-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| | | | | | | |
| :heavy_check_mark: | f0ee5df4 | [#11](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22009/11/) | 2025-02-05 15:18:35 | ~4 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250205-151339-f0ee5d-pr22009-tests.log) |
| :heavy_check_mark: | f0ee5df4 | [#11](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22009/11/) | 2025-02-05 15:22:21 | ~8 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250205-151335-f0ee5d-pr22009-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250205-151335-f0ee5d-pr22009-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | f0ee5df4 | [#11](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22009/11/) | 2025-02-05 15:23:00 | ~9 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-250205-151339-f0ee5d-pr22009-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-250205-151339-f0ee5d-pr22009-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | f0ee5df4 | [#11](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22009/11/) | 2025-02-05 15:27:16 | ~13 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/Eri2PV) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FEri2PV&size=400x400&errorCorrectionLevel=L)|

Parveshdhull commented:
~need to update current solution: https://github.com/status-im/status-mobile/pull/22008#discussion_r1938955067~ (upd: done)

Parveshdhull commented on src/status_im/contexts/profile/backup_recovery_phrase/view.cljs:
```diff
@@ -108,8 +109,7 @@
              {:actions          :one-action
               :button-one-label (i18n/label :t/i-have-written)
-              :button-one-props {:disabled?           (some false? (vals @checked?))
-                                 :customization-color customization-color
```
In the normal flow, `revealed?` can only be toggled once all checks are complete. Therefore, the `:disabled?` check is unnecessary.
For the `revealed?` true be default(see recovery phrase again flow), we don't care about checks.

flexsurfer commented on src/status_im/contexts/keycard/create/events.cljs:
```diff
@@ -31,6 +32,17 @@
           [:open-modal :screen/backup-recovery-phrase-dark
            {:on-success backup-recovery-phrase-success}]]]}))
 
+(rf/reg-event-fx :keycard/create.see-phrase-again
```
why this event is needed? double dispatch isn't good, could dispatch open modal directly

Parveshdhull committed:
Allow users to view the recovery phrase again when creating a profile for an empty keycard

status-im-auto commented:
## 55% of end-end tests have passed
```
Total executed tests: 11
Failed tests: 5
Expected to fail tests: 0
Passed tests: 6
```
```
IDs of failed tests: 727231,741555,741612,702843,741554 
```
<h3>Failed tests (5)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/keycard-see-recovery-phrase-again&PR_ID=22009&APK_NAME=22009.apk&TR_CASE_IDS=727231,741555,741612,702843,741554">Rerun failed tests</a></li><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777340">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><p><blockquote><div># STEP: Adding new regular account</div><div>Device 1: Find `Button` by `accessibility id`: `add-account` </div></blockquote></p>

```
critical/test_wallet.py:552: in test_wallet_add_remove_regular_account
    self.wallet_view.add_regular_account(account_name=new_account_name)
../views/wallet_view.py:215: in add_regular_account
    self.add_account_button.click()
../views/base_element.py:89: in click
    element = self.find_element()
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `add-account` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777343">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td><p><blockquote><div>Device 1: Could not reach Button by pressing system back button </div><div>Device 1: Find `Button` by `xpath`: `//android.view.ViewGroup[contains(@content-desc,'Account 1')]` </div></blockquote></p>

```
critical/test_wallet.py:349: in test_wallet_swap_flow_mainnet
    self.wallet_view.get_account_element().click()
../views/base_element.py:89: in click
    element = self.find_element()
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//android.view.ViewGroup[contains(@content-desc,'Account 1')]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777344">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td><p><blockquote><div>Device 1: Could not reach Button by pressing system back button </div><div>Device 1: Find `Button` by `xpath`: `//android.view.ViewGroup[contains(@content-desc,'Account 1')]` </div></blockquote></p>

```
critical/test_wallet.py:435: in test_wallet_bridge_flow_mainnet
    self.wallet_view.get_account_element().click()
../views/base_element.py:89: in click
    element = self.find_element()
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//android.view.ViewGroup[contains(@content-desc,'Account 1')]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777342">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
critical/test_wallet.py:337: in test_wallet_send_flow_mainnet
    self.wallet_view.slide_button_track.slide()
../views/base_view.py:107: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:288: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:274: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777336">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'Message AFTER edit 2 (Edited)')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-2']/android.widget.TextView[2]` </div><div>Device 1: Element EmojisNumber text is equal to 1 </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:388: in test_community_message_edit
    self.errors.verify_no_errors()
base_test_case.py:179: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 2: Message is not edited
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES5Pnj8qrsCZ8D9Kaq">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES6BmDEtR0VsA0R52I">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (6)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777335">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESkuYhWUY7GW2MPAfL">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES92iCOAuzsn3x8Nyg">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777337">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyZDpbU8cAR2tsNSW">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777334">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyZDpbU8cAR2tsNSW">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777341">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777339">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777338">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table></details>

status-im-auto commented:
## 45% of end-end tests have passed
```
Total executed tests: 11
Failed tests: 6
Expected to fail tests: 0
Passed tests: 5
```
```
IDs of failed tests: 727231,741555,703133,740490,741612,741554 
```
<h3>Failed tests (6)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/keycard-see-recovery-phrase-again&PR_ID=22009&APK_NAME=22009.apk&TR_CASE_IDS=727231,741555,703133,740490,741612,741554">Rerun failed tests</a></li><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777477">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><p><blockquote><div># STEP: Adding new regular account</div><div>Device 1: Find `Button` by `accessibility id`: `add-account` </div></blockquote></p>

```
critical/test_wallet.py:552: in test_wallet_add_remove_regular_account
    self.wallet_view.add_regular_account(account_name=new_account_name)
../views/wallet_view.py:215: in add_regular_account
    self.add_account_button.click()
../views/base_element.py:89: in click
    element = self.find_element()
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `add-account` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777480">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td><p><blockquote><div>Device 1: Could not reach Button by pressing system back button </div><div>Device 1: Find `Button` by `xpath`: `//android.view.ViewGroup[contains(@content-desc,'Account 1')]` </div></blockquote></p>

```
critical/test_wallet.py:349: in test_wallet_swap_flow_mainnet
    self.wallet_view.get_account_element().click()
../views/base_element.py:89: in click
    element = self.find_element()
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//android.view.ViewGroup[contains(@content-desc,'Account 1')]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777478">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `network-dropdown` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/test_wallet.py:264: in test_wallet_balance_mainnet
    self.errors.verify_no_errors()
base_test_case.py:179: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: For the Ether the wrong value 0.0362 is shown, expected 0.0262 in total
E    Device 1: For the Ether the wrong value 0.015 is shown, expected 0.005 on Mainnet
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777481">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td><p><blockquote><div>Device 1: Could not reach Button by pressing system back button </div><div>Device 1: Find `Button` by `xpath`: `//android.view.ViewGroup[contains(@content-desc,'Account 1')]` </div></blockquote></p>

```
critical/test_wallet.py:435: in test_wallet_bridge_flow_mainnet
    self.wallet_view.get_account_element().click()
../views/base_element.py:89: in click
    element = self.find_element()
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//android.view.ViewGroup[contains(@content-desc,'Account 1')]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777479">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
critical/test_wallet.py:337: in test_wallet_send_flow_mainnet
    self.wallet_view.slide_button_track.slide()
../views/base_view.py:107: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:288: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:274: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777474">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div># STEP: Check that removed user is not shown in the list anymore</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:246: in test_restore_multiaccount_with_waku_backup_remove_profile_switch
    self.errors.verify_no_errors()
base_test_case.py:179: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: zQ3...dWXh5 was not restored as a contact from waku backup!
E    Device 1: zQ3...Vacac was not restored as a contact from waku backup!
E    Device 1: admin_open was not restored from waku-backup!!
E    Device 1: member_open was not restored from waku-backup!!
E    Device 1: admin_closed was not restored from waku-backup!!
E    Device 1: member_closed was not restored from waku-backup!!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESXdt4OQhIolFTOSLZ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (5)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777471">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESXdt4OQhIolFTOSLZ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777472">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESu3XasyFmZOY4Qsbe">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESRw7uSmvr3WwWXtDw">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777476">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777475">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3777473">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESCD72TwxA7n3Dr5sU">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESDi2SKikdQN9xB5Of">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

status-im-auto commented:
## 91% of end-end tests have passed
```
Total executed tests: 11
Failed tests: 1
Expected to fail tests: 0
Passed tests: 10
```
```
IDs of failed tests: 702843 
```
<h3>Failed tests (1)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/keycard-see-recovery-phrase-again&PR_ID=22009&APK_NAME=22009.apk&TR_CASE_IDS=702843">Rerun failed tests</a></li><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3778055">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'Message AFTER edit 2 (Edited)')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-2']/android.widget.TextView[2]` </div><div>Device 1: Element EmojisNumber text is equal to 1 </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:388: in test_community_message_edit
    self.errors.verify_no_errors()
base_test_case.py:179: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 2: Message is not edited
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLt0pxMEzVM6oElck">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVqp6Irk5YMNEAXAU">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (10)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3778058">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3778057">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3778056">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShwMk3tOWjdnrT3bU">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3778053">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShwMk3tOWjdnrT3bU">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3778059">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3778062">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3778060">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3778063">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3778061">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3778054">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8zTgWYgaEOUjM0Ch">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyS9yIzaIch0ddOYB">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

pavloburykh commented:
@Parveshdhull thank you for the fix. Please take a look at the issue below. 

### ISSUE 1 "See recovery phrase again" button does not redirect user to the screen with revealed seed phrase

**Steps:** 

1. Generate drawer with "See recovery phrase again" button
2. Tap the button
3. See if user is redirected to the screen with revealed seed phrase

**Expected result:** user is redirected to the screen with revealed seed phrase

**Actual result:** user is not redirected to the screen with revealed seed phrase

Android:


https://github.com/user-attachments/assets/1bf27afc-7437-4112-81bc-fb0b8c0c279a



IOS:

https://github.com/user-attachments/assets/4891b122-6d05-4d95-a938-c35a92a1c802

Parveshdhull committed:
fix qa issues

Parveshdhull commented:
Hi @pavloburykh ,

Thank you very much for testing the PR.

I tested the fix when creating the PR, but I messed it up while addressing the review feedback. Apologies for that — it should be fixed now.

pavloburykh commented:
@Parveshdhull thank you for fixing ISSUE 1. Please take a look at the ISSUE 2 below. 

### ISSUE 2 "See recovery phrase again" button does not redirect user to the screen with revealed seed phrase _during Migration flow_

**Steps:** 

1. Create a new Profile (seedphrase should not be backed up yet)
2. Go to Profile - Keycard - initiate Keycard Migration process
3. Generate drawer with "See recovery phrase again" button
4. Tap the button
5. See if user is redirected to the screen with revealed seed phrase

**Expected result:** user is redirected to the screen with revealed seed phrase

**Actual result:** user is not redirected to the screen with revealed seed phrase


https://github.com/user-attachments/assets/d05d549e-13ea-46fc-b88a-2fb852b58827

Parveshdhull committed:
fix issue 2

Parveshdhull commented:
> ISSUE 2 "See recovery phrase again" button does not redirect user to the screen with revealed seed phrase during Migration flow

Thank you, should be fixed now.

pavloburykh commented:
@Parveshdhull thank you, ISSUE 2 is fixed. PR is ready for merge now. 

What has been tested: 

1. "See recovery phrase again" button redirects user back to the screen with revealed seedphrase. Checked behaviour on Onboarding (Create Profile) and Migration flows. 
2. Smoke checked navigation in different keycard flows works as before.
